src/utils/message.js

```diff
@@ -11,7 +11,7 @@ export const shouldBeMuted = (
   subscriptions: Subscription[] = [],
   mutes: MuteState = [],
 ): boolean => {
-  if (typeof message.display_recipient !== 'string') {
+  if (message.type === 'private') {
```

Comment:
I didn't got why this change is required?

Reply:
It isn't required, but as @gnprice [pointed](https://chat.zulip.org/#narrow/stream/48-mobile/topic/asserting.20type.20(flow)/near/722967) out, the code is cleaner and more readable this way.

>For runtime conditionals, what we most often do is look at a property with a name like type, whose value is one of a few fixed strings. We design the types so that knowing that value decides what other properties will exist with what types.

>In the case of Message, there's one of those with value either 'stream' or 'private'.

This also makes the tests more strict.